common/meson.build

```diff
@@ -9,6 +9,7 @@ lib_sway_common = static_library(
 		'pango.c',
 		'readline.c',
 		'stringop.c',
+		'unicode.c',
```

Comment:
Would it be possible to use mbrtoc32/c32rtomb from <uchar.h>, instead of doing it ourselves?

Reply:
No, it's not portable.

Reply:
They're standard C11 functions. I'm not certain, but they may introduce some locale-specific shit, though.